### ISSUE TITLE
[BUG] Extension verification fails on MacOS

### ISSUE THREAD

**Issue by nineinchnick** [tableau/connector-plugin-sdk#903]
**Describe the bug**
In 2021.3 running on MacOS 12, verification of the Starburst Enterprise extension fails.

Manual verification works:
```
% jarsigner -verify ~/Documents/My\ Tableau\ Repository/Connectors/StarburstEnterprise.taco         

jar verified.
% jarsigner -keystore /Applications/Tableau\ Desktop\ 2021.3.app/Contents/Plugins/jre/lib/security/cacerts -verify ~/Documents/My\ Tableau\ Repository/Connectors/StarburstEnterprise.taco          
jar verified.
```

The cert used to sign the taco is valid until 2023-09-11. I'm attaching output of `jarsigner -verbose -certs`.
[default.txt](https://github.com/tableau/connector-plugin-sdk/files/7667376/default.txt)

From `log.txt`:
```
{"ts":"2021-12-07T10:39:12.418","pid":90462,"tid":"49ac8a","sev":"error","req":"-","sess":"-","site":"-","user":"-","k":"connector-plugin-error","e":{"excp-error-code":"0x7D476C80","excp-source":"NeedsClassification","excp-status-code":"UNKNOWN"},"v":"Taco
 signature verification failed for /Users/jwas/Documents/My Tableau Repository/Connectors/StarburstEnterprise.taco \nExit code: 1.\n"}
{"ts":"2021-12-07T10:39:12.418","pid":90462,"tid":"49ac8a","sev":"warn","req":"-","sess":"-","site":"-","user":"-","k":"excp","e":{"excp-error-code":"0x14D18B1F","excp-source":"System","excp-status-code":"INVALID_ARGUMENT"},"v":{"class":"","dse-type":"Conn
ectorPluginFailure","excp-msg":"Package signature verification failed during connection creation.\n","excp-type":"ConnectivityException","is-bad-request":false,"is-capability-probe-failure":false,"is-local-configuration-error":true,"is-remote-configuration
-error":false,"msg":"ConnectivityException::Init"}}
```

**Desktop (please complete the following information):**
 - OS: MacOS 12.0.1
 - Tableau Version: 2021.3.4

**About you:**
 Name: Jan Waś
 Company: Starburst Data

**Comment by leniartek** [comment]
Cc. @VasanthiNeelagiri

**Comment by jie-d** [comment]
Internal tracking id: 1352820

**Comment by VasanthiNeelagiri** [comment]
Hi Jan & Team, 

Thanks fro raising the ticket and letting us know. This is a known issue being investigated internally. As you guys mentioned, using the previous maintenance release is a workaround. The issue is specific to Mac, so using Windows is another workaround. 

Thanks & Regards,
Vasanthi Neelagiri,
Product Manager, 
Tableau

**Comment by VasanthiNeelagiri** [comment]
The known issues page now is updated with this issue -(1352822) https://www.tableau.com/support/known-issues

Here's the KB Article for your reference- https://kb.tableau.com/articles/Issue/taco-based-connectors-validation-fails-on-mac-os

Thanks & Regards,
Vasanthi Neelagiri,
Product Manager,
Tableau

**Comment by pvanderknyff** [comment]
This bug is fixed has been fixed in the newly released December maintenance release: https://www.tableau.com/support/releases/desktop/2021.3.5

**Comment by leniartek** [comment]
@pvanderknyff was this issue also fixed for Tableau Desktop 2021.4.1 ?

**Comment by pvanderknyff** [comment]
This issue never affected the released version of 2021.4, just the maintenance releases. 2021.4.1 does not have this issue.
